src/main/java/at/hannibal2/skyhanni/utils/LorenzVec.kt

```diff
@@ -56,33 +57,54 @@ data class LorenzVec(
         return (dx * dx + dz * dz)
     }
 
+    operator fun plus(other: LorenzVec) = LorenzVec(x + other.x, y + other.y, z + other.z)
+    operator fun plus(other: Double) = LorenzVec(x + other, y + other, z + other)
+    operator fun plus(other: Int) = LorenzVec(x + other, y + other, z + other)
```

Comment:
not a fan of `vector + double` and `vector + int`
the normal functions are intentionally having default values, so that you can easily "just add y by 2" or somehtng.